packages/repository/src/repository.ts

```diff
@@ -113,24 +124,48 @@ export interface EntityCrudRepository<T extends Entity, ID>
    * @param data
    * @param id
    * @param options
+   * @promise {Boolean} resolves a Boolean
    */
   replaceById(id: ID, data: ObjectType<T>, options?: Options): Promise<boolean>;
 
   /**
    * Delete an entity by id
    * @param id
    * @param options
+   * @promise {Boolean} resolves a Boolean
    */
   deleteById(id: ID, options?: Options): Promise<boolean>;
 
   /**
    * Check if an entity exists for the given id
+   * @param id
+   * @param options
+   * @promise {Boolean} resolves a Boolean
```

Comment:
Shouldn't all the params have the type of the param on the docs specified with a small description? Something like this:

```
@param id {ID} The `id` value
```